src/plugins/kibana_usage_collection/server/plugin.ts

```diff
@@ -128,11 +127,6 @@ export class KibanaUsageCollectionPlugin implements Plugin {
 
     registerUiCountersUsageCollector(usageCollection, this.logger);
 
-    registerUsageCountersRollups(
```

Comment:
I moved this logic to the `usage_collection` plugin, IMO it makes more sense to have it there, WDYT?